### ISSUE TITLE
Support LPG creation and routing tables

### ISSUE THREAD

**Issue by slmjy** [oracle-terraform-modules/terraform-oci-oke#295]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Currently, there is no way to add LPG to the VCN and add routing to this LPG from workers subnet. A subnet can have multiple security lists, but routing table is only one. So effectively there is no way to connect the VCN with other VCNs in the same region.

### Potential Terraform Configuration

```hcl
module "lpg_connected_cluster" {
  source  = "oracle-terraform-modules/oke/oci"

  # this will automatically add lpgs and routing rules for the subnets
  # Ideally modify the security lists to
  create_lpgs = {
     "vcn_1" = {
         cidr = "..."
         peer_id= "OCID"
         peer_region_name = "...."
         subnets_to_route = [
            "workers"
         ]
      }
  }

```
those lpgs should also be in output of the module

**Comment by hyder** [comment]
Thanks for logging this issue @slmjy. We have an open issue in the upstream vcn module: https://github.com/oracle-terraform-modules/terraform-oci-vcn/issues/38

**Comment by slmjy** [comment]
@hyder Good to know, I actually forgot that it has dependency on upstream modules, luckily it was already logged there :)

**Comment by hyder** [comment]
closed in #332
